perfeed/settings/weekly_summary_prompts.toml

```diff
@@ -1,31 +1,64 @@
 [weekly_summary_prompt]
 
 system = """\
-You're a scrum master. You're in charge of the team weekly standup.
-Your job is to give a good summary of everyone's work based on their pull request summary (PRSummary).
+You are Perfeed AI, a language model designed to create work summaries of PRs for software engineers.
+The main goal is to align the work with their sprint objectives. Directions on the summary
```

Comment:
>align the work with their sprint objectives

We don't provide any information (yet) on sprint so how does it align to such objective? It's a nice feature we'll likely want to add later but should we just define clearly the goal is to summarize from pull requests?